README.md

```diff
@@ -27,41 +34,60 @@ gem "opencensus"
 $ bundle install
 ```
 
-## Tracing on Rack-based frameworks
-
-The OpenCensus library for Ruby makes it easy to integrate OpenCensus tracing
-into popular Rack-based Ruby web frameworks such as Ruby on Rails and
-Sinatra. When the library integration is enabled, it automatically traces
-incoming requests in the application.
+### Getting started with Ruby on Rails
 
-### With Ruby on Rails
+The OpenCensus library provides a Railtie that integrates with Ruby On Rails,
+automatically tracing incoming requests in the application. It also
+automatically traces key processes in your application such as database queries
+and view rendering.
 
-You can load the Railtie that comes with the library into your Ruby
-on Rails application by explicitly requiring it during the application startup:
+To enable Rails integration, require this file during application startup:
 
 ```ruby
 # In config/application.rb
 require "opencensus/trace/integrations/rails"
 ```
 
-If you're using the `opencensus` gem, it automatically loads the Railtie into
-your application when it starts.
-
-### With other Rack-based frameworks
+### Getting started with other Rack-based frameworks
 
 Other Rack-based frameworks, such as Sinatra, can use the Rack Middleware
-provided by the library:
+integration, which automatically traces incoming requests. To enable the
+integration for a non-Rails Rack framework, add the middleware to your
+middleware stack.
 
 ```ruby
-require "opencensus/trace"
+# In config.ru or similar Rack configuration file
+require "opencensus/trace/integrations/rack_middleware"
 use OpenCensus::Trace::Integrations::RackMiddleware
 ```
 
+## Instrumentation features
+
+### Tracing outgoing HTTP requests
+
+If you request uses the [Faraday](https://github.com/lostisland/faraday)
+library to make outgoing HTTP requests, consider installing the Faraday
+Middleware integration. This creates a span for each outgoing Faraday request,
+tracking the latency of that request, and it propagates distributed trace
+headers into the request so you can potentially connect your request trace with
+that of the remote service. Here is an example:
+
+```ruby
+conn = Faraday.new(url: "http://www.example.com") do |c|
+  c.use OpenCensus::Trace::Integrations::FaradayMiddleware
+  c.adapter Faraday.default_adapter
+  end
+conn.get "/"
+```
+
+See the documentation for the
+`OpenCensus::Trace::Integrations::FaradayMiddleware` class for more info.
```

Comment:
We are generating YARD docs so maybe add a link to http://opencensus.io/opencensus-ruby/api/OpenCensus/Trace/Integrations/FaradayMiddleware.html

Reply:
Done